docs/js/viewer.js

```diff
@@ -139,9 +139,11 @@ function redirectUrls(node) {
       continue;
     else if (href.startsWith('#'))
       addDynamicAnchorEvent(a); // on firefox, the second click on the anchor is not dealt cleanly
-    else if (href.startsWith('http')) // open external links in a new window
+    else if (href.startsWith('http')) {
+      // open external links in a new window
       a.setAttribute('target', '_blank');
-    else if (href.endsWith('.md') || href.indexOf('.md#') > -1) {
+      a.setAttribute('href', href.replaceAll('{version}', localSetup.branch));
```

Comment:
Note that other placeholders user the form `{{ <variable_name> }}` (e.g., `{{ webots.version.full }}`).
Won't be possible to use the same format? That would be more consistent and easier to use.

Maybe we could also replace the whole HTTPS URL prefix with the variable (`https://github.com/cyberbotics/webots/tree/<branch>`) given that it doesn't seem we need for anything else and this will simplify the MD files.

Reply:
I am on it. @omichel mentioned to me about the plugins, I completely overlooked it :/